docs/upctl_server_network-interface_modify.md

```diff
@@ -0,0 +1,35 @@
+## upctl server network-interface modify
+
+Modify a network interface
+
+```
+upctl server network-interface modify <UUID/Title/Hostname...> [flags]
+```
+
+### Options
+
+```
+      --index int                    Index of the interface to modify.
+      --new-index int                Index of the interface to modify.
+      --bootable string              Whether to try booting through the interface.
+      --source-ip-filtering string   Whether source IP filtering is enabled on the interface. Disabling it is allowed only for SDN private interfaces.
```

Comment:
looks like these two should be bools (tristate?) but are actually strings.. i guess the bool flags could really use some love before release?